### DISCUSSION TITLE
[processor/metricstransform] is it possible to combine multiple data points?

### DISCUSSION THREAD

**Discussion by alolage-devrev** [open-telemetry/opentelemetry-collector-contrib#10104] [Q&A]
Is it possible to use metricstransform processor to downsample metric data points? If yes, can someone provide a working sample processor config? 

for eg. I have 2 service instances serving /hello endpoint and pushing metrics to Otel as gateway. 

2022-05-16T23:02:51.205Z	INFO	loggingexporter/logging_exporter.go:56	MetricsExporter	{"#metrics": 1}
2022-05-16T23:02:51.205Z	DEBUG	loggingexporter/logging_exporter.go:66	ResourceMetrics #0
Resource SchemaURL:
Resource labels:
     -> service.name: STRING(unknown_service:api)
     -> telemetry.sdk.version: STRING(1.7.0)
     -> telemetry.sdk.language: STRING(go)
     -> telemetry.sdk.name: STRING(opentelemetry)
     -> revo: STRING(revo_2)
     -> api_path: STRING(/hello)
ScopeMetrics #0
ScopeMetrics SchemaURL:
InstrumentationScope
Metric #0
Descriptor:
     -> Name: helloCounter
     -> Description:
     -> Unit:
     -> DataType: Gauge
NumberDataPoints #0
StartTimestamp: 2022-05-16 19:16:32.334588121 +0000 UTC
Timestamp: 2022-05-16 23:02:42.334721598 +0000 UTC
Value: 58
NumberDataPoints #1
StartTimestamp: 2022-05-16 19:12:19.473998432 +0000 UTC
Timestamp: 2022-05-16 23:02:44.47412958 +0000 UTC
Value: 86

can above be converted into below in processor pipeline? 

2022-05-16T23:02:51.205Z	INFO	loggingexporter/logging_exporter.go:56	MetricsExporter	{"#metrics": 1}
2022-05-16T23:02:51.205Z	DEBUG	loggingexporter/logging_exporter.go:66	ResourceMetrics #0
Resource SchemaURL:
Resource labels:
     -> service.name: STRING(unknown_service:api)
     -> telemetry.sdk.version: STRING(1.7.0)
     -> telemetry.sdk.language: STRING(go)
     -> telemetry.sdk.name: STRING(opentelemetry)
     -> revo: STRING(revo_2)
     -> api_path: STRING(/hello)
ScopeMetrics #0
ScopeMetrics SchemaURL:
InstrumentationScope
Metric #0
Descriptor:
     -> Name: helloCounter
     -> Description:
     -> Unit:
     -> DataType: Gauge
NumberDataPoints #0
StartTimestamp: 2022-05-16 19:16:32.334588121 +0000 UTC
Timestamp: 2022-05-16 23:02:45.334721598 +0000 UTC                       << timestamp from end of batching period
Value: 144

**Comment by NeilLobo-kubra** [comment]
Did you resolve this?
